app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/MainActivity.java

```diff
@@ -6,30 +6,49 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
 import android.widget.Toast;
 
 import com.epfl.neighborfood.neighborfoodandroid.R;
 import com.epfl.neighborfood.neighborfoodandroid.ui.fragments.AccountFragment;
 import com.epfl.neighborfood.neighborfoodandroid.ui.fragments.ConversationsFragment;
 import com.epfl.neighborfood.neighborfoodandroid.ui.fragments.MealListFragment;
+import com.epfl.neighborfood.neighborfoodandroid.ui.fragments.VendorDashboardFragment;
 import com.google.android.material.navigation.NavigationBarView;
 
 public class MainActivity extends AppCompatActivity {
     public static final String EXTRA_MESSAGE = "com.epfl.neighborfood.MESSAGE";
     private NavigationBarView navbar;
+    private Button toggleButton;
+    private boolean isVendor;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         setCurrentFragment(MealListFragment.class);
         navbar =findViewById(R.id.bottomNavigationView);
         navbar.setOnItemSelectedListener(this::switchFragment);
+        isVendor=false;
+        toggleButton=findViewById(R.id.button2);
+        toggleButton.setOnClickListener(this::toggleView);
     }
+
+    private void toggleView(View view) {
```

Comment:
I would have specified in the method name the fact that it is the vendor/client view